stories/core/marginTypes.js

```diff
@@ -0,0 +1,15 @@
+export const all = [1, 0.5]
+export const bottom = [0, 0, 1]
+export const bottomHalf = [0, 0, 0.5]
+export const horizontal = [0, 1]
+export const horizontalHalf = [0, 0.5]
+export const item = [0, 0.5, 1]
+export const none = []
+export const right = [0, 1, 0, 0]
+export const top = [1, 0, 0, 0]
+export const topLeft = [1, 0, 0, 1]
+export const topRight = [1, 1, 0, 0]
+export const vertical = [1, 0]
+export const verticalHalf = [0.5, 0]
+export const xBottom = [1, 1, 0]
+export const xItem = [1, 0.5, 0]
```

Comment:
hmm, its not super descriptive. But I'm not sure I have a better alternative. Maybe "AntiBottom"? "ReverseBottom"?